modules/databend/databend.go

```diff
@@ -0,0 +1,147 @@
+package databend
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	databendUser        = "databend"
+	defaultUser         = "databend"
+	defaultPassword     = "databend"
+	defaultDatabaseName = "default"
+)
+
+// DatabendContainer represents the Databend container type used in the module
+type DatabendContainer struct {
+	testcontainers.Container
+	username string
+	password string
+	database string
+}
+
+var _ testcontainers.ContainerCustomizer = (*DatabendOption)(nil)
+
+// DatabendOption is an option for the Databend container.
+type DatabendOption func(*DatabendContainer)
+
+// Customize is a NOOP. It's defined to satisfy the testcontainers.ContainerCustomizer interface.
+func (o DatabendOption) Customize(*testcontainers.GenericContainerRequest) error {
+	// NOOP to satisfy interface.
+	return nil
+}
+
+// Run creates an instance of the Databend container type
+func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (*DatabendContainer, error) {
+	req := testcontainers.ContainerRequest{
+		Image:        img,
+		ExposedPorts: []string{"8000/tcp"},
+		Env: map[string]string{
+			"QUERY_DEFAULT_USER":     defaultUser,
+			"QUERY_DEFAULT_PASSWORD": defaultPassword,
+		},
+		WaitingFor: wait.ForAll(
```

Comment:
suggestion: remove the ForAll as it's not needed

Reply:
Make sense. Fixed.